packages/twenty-front/src/modules/activities/emails/components/EmailThreadPreview.tsx

```diff
@@ -100,7 +104,13 @@ export const EmailThreadPreview = ({
         ];
 
   return (
-    <StyledCardContent onClick={() => onClick()} divider={divider}>
+    <StyledCardContent
+      onClick={() => onClick()}
+      divider={divider}
+      style={
```

Comment:
Can we pass a visibility props to the styled component instead on inline css here?